esque/cli/commands.py

```diff
@@ -328,59 +322,190 @@ def get_topics(state: State, prefix: str, output_format: str):
     click.echo(format_output(topic_names, output_format))
 
 
-@esque.command("transfer", help="Transfer messages of a topic from one environment to another.")
+@esque.command("consume", help="Consume messages of a topic from one environment to a file or STDOUT")
 @click.argument("topic", required=True)
-@click.option("-f", "--from", "from_context", help="Source Context", type=click.STRING, required=True)
-@click.option("-t", "--to", "to_context", help="Destination context", type=click.STRING, required=True)
-@click.option("-n", "--numbers", help="Number of messages", type=click.INT, required=True)
-@click.option("-m", "--match", help="Match expression", type=click.STRING, required=False)
+@click.option("-f", "--from", "from_context", help="Source Context", type=click.STRING, required=False)
+@click.option("-n", "--numbers", help="Number of messages", type=click.INT, default=sys.maxsize, required=False)
+@click.option("-m", "--match", help="Message filtering expression", type=click.STRING, required=False)
 @click.option("--last/--first", help="Start consuming from the earliest or latest offset in the topic.", default=False)
 @click.option("-a", "--avro", help="Set this flag if the topic contains avro data", default=False, is_flag=True)
 @click.option(
-    "-k",
-    "--keep",
-    "keep_file",
-    help="Set this flag if the file with consumed messages should be kept.",
+    "-o",
```

Comment:
```suggestion
    "-p",
```
Other commands use `-o`  for `--output-format` which could lead to confusion.

Reply:
As with the consume command, better to leave out the short option completely and then implement a "-" as a substitute for stdin/stdout.